spec/lib/vmdb/settings_spec.rb

```diff
@@ -106,6 +106,18 @@
       )
     end
 
+    it "with a change to an Array" do
+      array_hash = {:log => {:collection => {:current => {:pattern => ["*.log"]}}}}
+      described_class.save!(miq_server, array_hash)
```

Comment:
Same as below, it wasn't obvious that this depends on the current state of the settings.yml's `pattern` value being an array.